### PR TITLE
SWC-6162 - Properly configure QueryClient

### PR DESCRIPTION
**Pull request by nickgros**
Note: base branch is release-410 (staging)

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/DefaultQueryClientOptions.java [hunk 1]
```diff
@@ -7,12 +7,12 @@
 
 @JsType(isNative = true, namespace = JsPackage.GLOBAL, name="Object")
 public class DefaultQueryClientOptions {
-    public QueryOptions query;
+    public QueryOptions queries;
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/DefaultQueryClientOptions.java:10]: The field is called `queries`, so the settings never got set!

File: src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/QueryOptions.java [hunk 2]
```diff
@@ -6,13 +6,12 @@
...
-    public Long staleTime;
+    public long staleTime;
     public boolean retry;
 
     @JsOverlay
     public static QueryOptions create() {
         QueryOptions queryOptions = new QueryOptions();
-        // We don't currently need to configure these values
         queryOptions.staleTime = 30 * 1000L; // 30s
         queryOptions.retry = false; // SynapseClient knows which queries to retry
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/QueryOptions.java:16]: Bug described in SWC-6162 was caused by retry defaulting to true, so the failed query requests would retry 3 times

File: src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/QueryOptions.java [hunk 3]
```diff
@@ -6,13 +6,12 @@
 
 @JsType(isNative = true, namespace = JsPackage.GLOBAL, name="Object")
 public class QueryOptions {
-    public Long staleTime;
+    public long staleTime;
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/jsinterop/reactquery/QueryOptions.java:9]: In jsinterop, wrapper `Long` maps to an object, primitive `long` maps to a number

**Review by jay-hodgson — APPROVED** [review]

**Comment by jay-hodgson** [comment]
Nice sleuthing @nickgros !

**Pull request merged**